src/content/basics/giant-swarm-support/index.md

```diff
@@ -17,11 +17,11 @@ We accompany our customers throughout their journey. Having someone to contact w
 
 Our customer support relies on close interactions via Slack, to get bi-directional feedback as quickly as possible. This is our first line of contact to customers. This interaction channel is also used to ask all kinds of questions. The conversations are not limited to the platform itself, but can go broader and include the whole Cloud Native journey.
 
-Our support shifts go from 08:00 to 18:00 on weekdays, but we also often answer questions well outside those times. We rotate support shifts across the team, which is why focus on support channels with clear internal handovers.
+Our support shifts go from 08:00 to 18:00 on weekdays, but we also often answer questions well outside those times. We rotate support shifts across the team, which is why why focus on support channels with clear internal handovers.
```

Comment:
Apart from the double why instead of why we, it is a nice set of changes

Reply:
I just got my environment set up, so should be getting better at proofreading. Thanks for having my back.